apps/avifenc.c

```diff
@@ -349,6 +356,52 @@ static avifBool readEntireFile(const char * filename, avifRWData * raw)
     return AVIF_TRUE;
 }
 
+static avifBool avifCodecSpecificOptionsAdd(avifCodecSpecificOptions * options, const char * keyValue)
```

Comment:
Looks like there's a mix of functions with an `avif` prefix that are static in this file. For this case I guess it was to match the parameter type like with `avifInput...`.